htmd/mdengine/acemd/acemd.py

```diff
@@ -692,25 +732,27 @@ def test_acemd2(self):
         import shutil
         shutil.rmtree(tmpdir)
 
-    def test_acemd3(self):
+    def test_acemd(self):
         import htmd.home
-        homedir = htmd.home.home()
+        homedir = htmd.home.home(dataDir='dhfr')
 
         r = list()
         r.append(GroupRestraint('resname MOL', 5, [(10, '10ns'), (5, '15ns'), (0, '20ns')], axes='z'))
-        r.append(GroupRestraint('resname MOL', 5, [(10, '10ns'), (5, '15ns'), (0, '20ns')], axes='z', fbcentre=[4, 2, 7.3]))
-        r.append(GroupRestraint('resname MOL', 5, [(10, '10ns'), (5, '15ns'), (0, '20ns')], axes='z', fbcentresel='protein'))
+        r.append(GroupRestraint('resname MOL', 5, [(10, '10ns'), (5, '15ns'), (0, '20ns')], axes='z',
```

Comment:
I'd prefer if we could separate formatting commits from actual changes. It makes diffs very hard to read :(
Same for me of course since I forget to do it

Reply:
I try my best at it. This commit is here: https://github.com/Acellera/htmd/pull/707/commits/e931ebfc87cc49383adac8f07ee63248a3aea8d1 and I think it's only formatting